### PR TITLE
[DO-2077] Add search filter to export xlsx

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: data_ocean/export.py [hunk 1]
```diff
@@ -18,9 +20,13 @@
 class ExportToXlsx:
 
     @staticmethod
-    def export(params, export_dict, model_path, filterset_module, serializer_module, user_id):
+    def export(params, export_dict, model_path, filterset_module, serializer_module, view_module, user_id):
         app, model = model_path.split('.')
         queryset = apps.get_model(app, model).objects.all()
+        view = import_string(view_module)
+        request = requests
```
Comment (roman-tiukh) [inline data_ocean/export.py:27]: use native request object from django DRF

File: data_ocean/export.py [hunk 2]
```diff
@@ -18,9 +20,13 @@
...
+        setattr(request, 'query_params', params)
+        queryset = SearchFilter.filter_queryset(SearchFilter(), request, queryset, view)
         queryset = import_string(filterset_module)(params,  queryset).qs
         serializer = import_string(serializer_module)(queryset, many=True)
```
Comment (roman-tiukh) [inline data_ocean/export.py:31]: you can omit serializer, it redundant

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: data_ocean/export.py (hunk 1 above)
Comment (Iva-khar) [reply data_ocean/export.py:27]: done

File: data_ocean/export.py (hunk 2 above)
Comment (Iva-khar) [reply data_ocean/export.py:31]: I need serializer for `pep_type_display` field, that generates in serializer

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**